README.md

```diff
@@ -422,6 +422,13 @@ func handle(payment: PKPayment) {
 ```
 
 
+### Phone number validation
+
+Billing address phone number validation will use the device local to set the prefix of the phone number. For example, a UK number will be automatically prefixed with +44.
+
+If users want to enter a phone number that differs from their device local (i.e. a US number when their device local is set up for the UK), they should first provide the country code (e.g. +1) when entering the phone number.
```

Comment:
This is for the case of entering a phone number from a different country than what your device is setup to be. 

You can always enter a number, but if there is no country code it will only use the country code of your device locale. So if user wants to enter foreign phone number they need to specify which country will the phone number belong to (unrelated to Country option on billing).